include/geode/geometry/basic_objects/segment.h

```diff
@@ -82,7 +81,7 @@ namespace geode
         OwnerSegment< dimension >& operator=(
             OwnerSegment< dimension >&& other );
     };
-    ALIAS_2D_AND_3D( OwnerSegment );
+    ALIAS_1D_AND_2D_AND_3D( OwnerSegment );
```

Comment:
Is direction a method of Segment1D, what does it mean?

Reply:
Not sure.. Probably the direction on the segment like +1 or - 1